src/components/previewArticle/Preview.jsx

```diff
@@ -12,7 +12,16 @@ const Preview = (props) => {
   const {
     onSubmit, article, continueEdit, loading
   } = props;
-  const image = article.image ? URL.createObjectURL(article.image) : 'https://via.placeholder.com/700x400';
+
+  let image;
```

Comment:
@chingsley this logic seems to be repeated in several places. If this is so, it should ring a bell of abstraction. If this ends up being abstracted into a function, make sure the new function replaces all the instances of this logic in the app

Reply:
Alright. I will do that.